client/stylesheets/_toCat.scss

```diff
@@ -2461,8 +2514,17 @@ i:hover {
 
 .headerDropDown ul li a:hover {
     background-color: #555;
-    transition: none !important;
-    
+    transition: none !important;   
+}
+
+.headerBigCat {
+	border: 1px solid #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs